dom.bs

```diff
@@ -5875,19 +5875,19 @@ dictionary ShadowRootInit {
 initialized.
 
 <p>An <a for=/>element</a>'s <a for=Element>custom element state</a> is one of
-"<code>undefined</code>", "<code>failed</code>", "<code>uncustomized</code>", or
-"<code>custom</code>". An <a for=/>element</a> whose <a for=Element>custom element state</a> is
-"<code>uncustomized</code>" or "<code>custom</code>" is said to be
-<dfn export id=concept-element-defined for=Element>defined</dfn>. An <a for=/>element</a> whose
-<a for=Element>custom element state</a> is "<code>custom</code>" is said to be
-<dfn export id=concept-element-custom for=Element>custom</dfn>.
+"<code>undefined</code>", "<code>failed</code>", "<code>uncustomized</code>",
+"<code>precustomized</code>", or "<code>custom</code>". An <a for=/>element</a> whose
+<a for=Element>custom element state</a> is "<code>uncustomized</code>" or "<code>custom</code>"
+is said to be <dfn export id=concept-element-defined for=Element>defined</dfn>. An
+<a for=/>element</a> whose <a for=Element>custom element state</a> is "<code>precustomized</code>"
+or "<code>custom</code>" is said to be <dfn export id=concept-element-custom for=Element>custom</dfn>.
```

Comment:
I don't think it's right to consider "precustomized" state as **custom** either.
This would mean that in the upgrade case, you can insert the node into a document tree before `super()` is the called, and the spec text will try to look for its **custom element definition** and fail, which would be self contradictory.

Reply:
Hmm, I added this in response to a comment from @annevk on [DOM-893](https://github.com/whatwg/dom/pull/893). How would you recommend writing [this check](https://github.com/whatwg/dom/pull/893/files#diff-a467e681501e456ce8c1ef31425e1b41R6744), which is meant to check for "precustomized" or "custom". As @annevk pointed out, the definition of "custom element" is "custom element state is custom". I would assume you would just prefer that check (in 893) to say this?

> If this's custom element state is "precustomized" or "custom", set shadow's **available to element internals** property to true. Otherwise, set it to false.

?

Reply:
It seems like we should just check that custom element state is either "custom" or "precusotmized"?

Reply:
Ok, done, both here and in [893](https://github.com/whatwg/dom/pull/893/files).